### ISSUE TITLE
Correção de conteúdo

### ISSUE THREAD

**Issue by Guilherme07** [desenvolvimentoagilcombr/desenvolvimentoagilcombr.github.com#45]
Favor, reconsiderar a nomeação de Scrum Team como apenas o time de desenvolvimento.
Referência do Scrum, o Scrum Guide deixa claro que o Time Scrum é composto pelo Time de Desenvolvimento, Product Owner e Scrum Master.

**Comment by ThiagosLima** [comment]
De acordo com o Scrum Guide - 2016, página 6, o time de desenvolvimento tem entre 3 e 9 pessoas. Os papéis de Scrum Master e Product Owner não são inclusos nessa contagem, a menos que eles executem o trabalho de desenvolvimento também.

**Comment by Guilherme07** [comment]
Exato, mas o Time Scrum não é a mesma coisa que o Time de desenvolvimento. O Time de Desenvolvimento compõe o Time Scrum, somado ao Scrum Master e Product Owner. No site, tratam o time scrum como time de desenvolvimento de forma equivocada
http://www.desenvolvimentoagil.com.br/scrum/scrum_team
